{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/data/loader.py

```diff
@@ -86,17 +85,19 @@ def load_theoretical_minimum_risk_life_expectancy(key: str, location: str) -> pd
     return interface.get_theoretical_minimum_risk_life_expectancy()
 
 
-def load_standard_data(key: str, location: str) -> pd.DataFrame:
+def load_standard_data(
+    key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
```

Comment:
Shouldn't we also add the years argument everywhere `load_standard_data` is called and all the way up to the `make_artifact` cli?